client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -217,6 +217,17 @@ UndocumentedSite.prototype.newExport = function( fn ) {
 };
 
 /**
+ * Returns media storage limits and space used for a given site. If site has
+ * unlimited storage or is a jetpack site, values returned will be -1.
+ *
+ * @param {Function} callback - called on completion of the GET request
+ * @return {Object} promise - resolves on completion of the GET request
+ */
+UndocumentedSite.prototype.getMediaStorage = function( callback ) {
```

Comment:
>Sure, `mediaStorage` then?

That's what I would have expected, yea.

>What's the naming convention for an endpoint with multiple actions, eg: GET/POST/DELETE/PUT

I may defer some to @retrofox on this one. You can see some variety in how these functions are named ([refer to docs](https://github.com/Automattic/wpcom.js/tree/master/docs)). In a case like this where there is no update action, `mediaStorage` seems appropriate. Otherwise, I might expect `mediaStorage()` to further add to the chain, on which you could call `mediaStorage().get()` or `mediaStorage().update()`